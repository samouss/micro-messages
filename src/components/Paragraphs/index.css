.Paragraphs {
  white-space: pre-line;
}
