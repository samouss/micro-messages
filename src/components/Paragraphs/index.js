// @flow

import React from 'react';
import './index.css';

export type Props = {
  text: string;
};

const Paragraphs = ({ text }: Props) => (
  <p styleName="Paragraphs">
    {text}
  </p>
);

export default Paragraphs;
