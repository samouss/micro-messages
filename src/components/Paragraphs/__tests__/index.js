// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Paragraphs from '../index';

import type { Props } from '../index';

describe('<Paragraphs />', () => {
  const defaultProps: Props = {
    text: 'Text from Lorem',
  };

  it('expect to render the component correctly', () => {
    const props: Props = {
      ...defaultProps,
    };

    const component = shallow(
      <Paragraphs
        {...props}
      />,
    );

    expect(component).toMatchSnapshot();
  });
});
