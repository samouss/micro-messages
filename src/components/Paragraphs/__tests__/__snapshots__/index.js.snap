// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraphs /> expect to render the component correctly 1`] = `
<p
  styleName="Paragraphs"
>
  Text from Lorem
</p>
`;
